src/test-support/mocks/src/dataplanes/_overview.ts

```diff
@@ -1,41 +1,64 @@
 import type { EndpointDependencies, MockResponder } from '@/test-support'
+
 export default ({ fake, pager, env }: EndpointDependencies): MockResponder => (req) => {
+  const query = req.url.searchParams
+  const _gateway = query.get('gateway') ?? ''
+  const _name = query.get('name') ?? ''
+  const _tags = query.get('tags') ?? ''
+
+  const inbounds = parseInt(env('KUMA_DATAPLANEINBOUND_COUNT', `${fake.number.int({ min: 1, max: 3 })}`))
   const subscriptionCount = parseInt(env('KUMA_SUBSCRIPTION_COUNT', `${fake.number.int({ min: 1, max: 10 })}`))
+  const isMtlsEnbaled = env('KUMA_MTLS_ENABLED', fake.helpers.arrayElement(['false', 'true'])) === 'true'
```

Comment:
nit: Little typo here, in both places, but still a typo

Reply:
Fixed.